content/tutorials/generic/bootloader-reloading/bootloader-reloading.md

```diff
@@ -0,0 +1,112 @@
+- An SD card that fits your SD slot.
+- A way to connect the SD card to your computer.
+- A way to make the connections to the SWD pins on your target Arduino board.
+
+For the [Nano 33 IoT](https://store.arduino.cc/products/arduino-nano-33-iot?_gl=1%2A80ta1j%2A_ga%2AMjEyMzQ2MjgwOC4xNjY1NjUyNTY3%2A_ga_NEXN8H46L5%2AMTY3MTYyNzMzMS4xNjEuMS4xNjcxNjMwNDIwLjAuMC4w) and the MKR Boards other than MKR1000, we found it the easiest to use a 0.1" pitch 2x3 POGO adapter. You could also solder wires to the test points if you prefer. On the MKR boards other than the MKR1000, the SWD header is on the bottom of the board and is the footprint for a 0.1" pitch 2x3 SMD header. On the MKR1000, it is a 0.05" pitch 2x5 male header on the top of the board, which you will need an adapter and one cable for.
+
+## Instructions
+- Connect an SD card to your computer.
+- Open this link in your browser: https://github.com/arduino/ArduinoCore-samd/tree/master/bootloaders.
+- Click the folder that matches the name of your target board.
+- Click the file that ends in ```.bin```.
+- Click the **Download** button.
+- Rename the downloaded file to ```fw.bin```.
+- Move ```fw.bin``` to the SD card.
+- Eject the SD card from your computer.
+- Plug the USB cable of the Arduino board you will be using as a programmer into your computer.
+- Select **Sketch > Include Library > Manage Libraries...** from the Arduino IDE's menus.
+- Wait for the download to finish.
+- In the **"Filter your search..."** field, type "Adafruit DAP library".
+- Press **Enter**.
+- Click on **"Adafruit DAP library by Adafruit"**.
+- Click the **"Install"** button.
+- Wait for the installation to finish.
+- Click the **Close** button.
+- Select **File > Examples > Adafruit DAP library > samd21 > flash_from_SD** from the Arduino IDE's menus.
+- Change this line: ```#define SD_CS 4``` according to the Arduino pin connected to the SD CS pin. 
+
+    If your board has a built-in SD slot (e.g., [MKR Zero 9](https://store.arduino.cc/products/arduino-mkr-zero-i2s-bus-sd-for-sound-music-digital-audio-data)), then you can change this line:
+
+    ```
+    (!SD.begin(SD_CS)) { 
+    ```
+
+    to:
+
+    ```
+    if (!SD.begin()) {
+    ```
+
+- Select the correct board from the **Tools > Board** from the Arduino IDE's menus.
+- Select the correct port from the **Tools > Port** from the Arduino IDE's menus.
+- Select **Sketch > Upload** from the Arduino IDE's menus.
+- Wait for the upload to finish successfully.
+- Unplug the programmer Arduino board from your computer.
+- Plug the SD card into the SD slot connected to your Arduino board.
+- Connect the programmer Arduino board to the target Arduino board as follows:
+
+    | Programmer  | Target      |
+    | ----------- | ----------- |
+    | VCC         | +3V3        |
+    | 10          | SWDIO       |
+    | 9           | SWCLK       |
+    | GND         | GND         |
+    | 11          | RESETN      |
```

Comment:
```suggestion
    | 12          | SWDIO       |
    | 11          | SWCLK       |
    | GND         | GND         |
    | 9           | RESETN      |
```

They changed the pins in the example sketch since the time I wrote that forum post:

https://github.com/adafruit/Adafruit_DAP/blob/1.7.4/examples/samd21/flash_from_SD/flash_from_SD.ino#L6-L8